kyaml/openapi/kustomizationSwagger.json

```diff
@@ -0,0 +1,442 @@
+{
+  "definitions": {
```

Comment:
This should include a definition for the Kustomization `Kind`

Reply:
Added